prestoadmin/__init__.py

```diff
@@ -17,30 +17,89 @@
 
 __version__ = '1.2'  # Make sure to update setup.py too
 
+
 import os
+import sys
+
+main_dir = os.path.dirname(os.path.abspath(os.path.dirname(__file__)))
 
 from fabric.api import env
 
+import fabric_patches
+import mode
+
+from prestoadmin.mode import get_mode, for_mode, MODE_STANDALONE, MODE_SLIDER
+from prestoadmin.util.exception import ConfigFileNotFoundError, \
+    ConfigurationError
+
+
+#
+# Subcommands common to all modes. If anybody knows why fabric_patches is in
```

Comment:
I believe it's imported so that everything in the prestoadmin module has fabric_patches loaded so the function calls use the patched version of all the functions.  It's not really an unused import.  Lint just doesn't know that it's really necessary.  I believe you can configure lint not to care about unused imports in __init__.py, or there is some annotation to say ignore warning for this.